pkg/scheduler/algorithm/predicates/utils.go

```diff
@@ -147,3 +148,193 @@ func isCSIMigrationOn(csiNode *storagev1beta1.CSINode, pluginName string) bool {
 
 	return mpaSet.Has(pluginName)
 }
+
+// utilities for building pod/node objects using a "chained" manner
+type nodeSelectorWrapper struct{ v1.NodeSelector }
+
+func makeNodeSelector() *nodeSelectorWrapper {
+	return &nodeSelectorWrapper{v1.NodeSelector{}}
+}
+
+// NOTE: each time we append a selectorTerm into `s`
+// and overall all selecterTerms are ORed
```

Comment:
you can just say "appended selectorTerms are ORed"

Reply:
Done.